css/style.css

```diff
@@ -1,32 +1,38 @@
 #container {
```

Comment:
Hi @2bleO👐,
Your project looks good! However, There are some changes you can perform to make it look better!

Good:
Correct linter configuration ✔️
Good Pull Request description ✔️

To Change:

- [ ] Please try using semantic html tags such as header instead of a div with a "header" class. This change can improve your SEO and overall user experience.

- [ ] Please refer to this Template to make your README more professional by adding information relevant to your current project.